### PR TITLE
Additional build fixes

### PR DESCRIPTION
**Pull request by CarlosNZ**
This will need to be tested properly. I can confirm that the build process should work fine and the docker image runs locally okay. There's still some weirdness with files not showing up correctly and some other things which I'll look into and maybe push to this PR too.

**Update**: When running the docker image locally, the core_templates snapshot hasn't fully loaded properly, although it's there enough to log in. So I can manually import a snapshot (including core_templates) and load that which works. But now I get this error when trying to start a new "User Registration"
Screen Shot 2021-11-10 at 11 54 08 AM

Inspecting the network logs, it's failing when trying to access `http://localhost:3000/postgraphile/graphql/api?dev=getTemplates` This doesn't seem right, but not sure why it's requesting that.

### CODE REVIEW

File: docker/run.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@
 docker run \
     -ti \
     -p 3000:3000 \
-    -e 'SMTP_PASSWORD=smtp_password_goes_here' \
+    -e 'SMTP_PASSWORD=smtp_password_goes_here' -e 'WEB_HOST=http://localhost:3000' \
```
Comment (nmadruga) [inline docker/run.sh:8]: Is this always the case? Well, I think that's only used to test locally right?

File: utils/build_all.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@ cp -R database/buildSchema ./build/database/
 cp -R database/insertData ./build/database/
 cp -R database/snapshotOptions ./build/database/
 cp -R database/core_templates ./build/database/
-echo 'copyling localisation files'
+echo 'copying localisation files'
```
Comment (nmadruga) [inline utils/build_all.sh:14]: Oh noe! And I locked to that for so long. Didn't see the typo! Good catch

File: documentation/Demo-Server-Guide.md [hunk 3]
```diff
@@ -160,6 +160,7 @@ sudo chown 472 grafana_on_port_8009
 export TAG='front-demo-19-08-2021_back-demo-19-08-2021_pg-12_node-14'
 export SMTP_SECRET='add_smtp_secret_here'
 export WEB_URL='https://irims-demo.msupply.org:<replace port>'
+export JWT_SECRET='random private key'
```
Comment (nmadruga) [inline documentation/Demo-Server-Guide.md:163]: Is this where we set the new JWT for this server? Maybe worth mentioning how to generate this JWT and use here. Do we need to keep this saved somewhere for future use? Like a Bitwarden entry?

**Review by nmadruga — CHANGES_REQUESTED** [review]
Just some thoughts

**Review by nmadruga — CHANGES_REQUESTED** [review]
Hi, sorry I was supposed to have sent this earlier to you

File: docker/run.sh (hunk 1 above)
Comment (CarlosNZ) [reply docker/run.sh:8]: Yes, exactly, you'll still need to change it for the server. This is just for running the docker image locally

File: documentation/Demo-Server-Guide.md (hunk 3 above)
Comment (CarlosNZ) [reply documentation/Demo-Server-Guide.md:164]: I don't think it really matters, we can just make up a new one every time we launch a new instance (or even restart the server). The only time we need to keep this value is if we want existing JWTs to remain valid (like in our testing environment, or with existing Postman collections, say)

But yes, have added a note 084b0c65

**Review by nmadruga — APPROVED** [review]

**Pull request merged**